dist/docs_data.py

```diff
@@ -0,0 +1,25 @@
+# Create entries used by our doxygen filter to expand the arch_page
+# macros in the documentation.
+
+class ArchDocPage:
+    def __init__(self, doxygen_name, data_structures, files):
+        self.doxygen_name = doxygen_name
+        self.data_structures = data_structures
+        self.files = files
+
+##########################################
+# List of all architecture subsections
+##########################################
+arch_doc_pages = [
```

Comment:
All of these things should be alphabetized I think.

Reply:
I will alphabetize the page list.

@sueloverso Do you think the list of data structures and the list of files should be alphabetized for each page?  These can all be enforced as well (we are already checking for file existence).

Reply:
I am always a fan of alphabetizing so, in general, yes. But it could be that we list `src/include` files first, and then source files below the header files. If we did that, they should be separately sorted. But if it is easier, then just alphabetizing the whole list works for me.

Reply:
Everything is alphabetized (but with header files first).  This is enforced by the `s_docs` script.